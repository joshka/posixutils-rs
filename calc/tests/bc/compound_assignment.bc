a = 4
a += 3.47
a
a -= 9.890
a
a *= 2
a
a /= 1
a
a %= 3
a
a ^= -2
a
quit
