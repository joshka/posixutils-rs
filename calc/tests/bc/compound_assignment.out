7.47
-2.420
-4.840
-4
-1
1
